docs: multi project (#81)
lacework/terraform-gcp-audit-log: examples/project-level-audit-log-multi/README.md

# Integrate Multiple GCP Project with Lacework
The following provides an example of integrating multiple Google Cloud Project with Lacework for Cloud Audit Log analysis.

The fields required for this example are:

| Name       | Description                                                                                 | Type     |
|------------|---------------------------------------------------------------------------------------------|----------|

> | `projects` | The id of each Project, which will be used to deploy required resources for the integration | `map` |

Reviewer: ```suggestion
| `projects` | Map of projects which will be used to deploy required resources for each integrations | `map` |
```

```hcl
provider "google" {
  project = keys(var.projects)[0]
}

provider "lacework" {}

variable "projects" {
  description = "Map of project configuration with Lacework."
  type        = map
  default     = {
    project-id-1 = "first project",
    project-id-2 = "second project"
  }
}

module "gcp_audit_log" {

>   source = "../../"

Reviewer: Use real source information for documentation:

```suggestion
  source = "lacework/audit-log/gcp"
  version = "~> 3.0"
```

  for_each   = var.projects
  project_id = each.key
}
```

Run Terraform:
```
$ terraform init

> $ GOOGLE_CREDENTIALS=account.json GOOGLE_PROJECT=my-project terraform apply

Reviewer: The project is already set in the provider block
```suggestion
$ GOOGLE_CREDENTIALS=account.json terraform apply
```

> # Integrate Multiple GCP Project with Lacework

Reviewer 2: ```suggestion
# Integrate Multiple GCP Projects with Lacework
```

> The following provides an example of integrating multiple Google Cloud Project with Lacework for Cloud Audit Log analysis.

Reviewer 2: ```suggestion
The following provides an example of integrating multiple Google Cloud projects with Lacework for cloud Audit Log analysis.
```

The fields required for this example are:

| Name       | Description                                                                                 | Type     |
|------------|---------------------------------------------------------------------------------------------|----------|

> | `projects` | Map of projects which will be used to deploy required resources for each integrations | `map` |

Reviewer 2: ```suggestion
| `projects` | Map of projects that will be used to deploy required resources for each integration | `map` |
```

```hcl
provider "google" {
  project = keys(var.projects)[0]
}

provider "lacework" {}

variable "projects" {
  description = "Map of project configuration with Lacework."
  type        = map
  default     = {
    project-id-1 = "first project",
    project-id-2 = "second project"
  }
}

module "gcp_audit_log" {
  source = "lacework/audit-log/gcp"
  version = "~> 3.0"

  for_each   = var.projects
  project_id = each.key
}
```

Run Terraform:
```
$ terraform init
$ GOOGLE_CREDENTIALS=account.json terraform apply
```

> For detailed information on integrating Lacework with Google Cloud see [GCP Compliance and Audit Trail Integration - Configure Multiple Porjects with Terraform](https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform)

Reviewer 2: ```suggestion
For detailed information on integrating Lacework with Google Cloud see [GCP Compliance and Audit Trail Integration - Configure Multiple Projects with Terraform](https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform)
```
